README.md

```diff
@@ -575,12 +581,12 @@ some kind of maximum queue depth for pre-connection commands.
 
 ## client.retry_delay
 
-Current delay in milliseconds before a connection retry will be attempted. This starts at `250`.
+Current delay in milliseconds before a connection retry will be attempted. This starts at `200`.
```

Comment:
I think starting with a lower delay might be good and we might actually also consider lowering the multiplier. What do you all think?